### ISSUE TITLE
Gradle version issue

### ISSUE THREAD

**Issue by deep1931** [mattermoran/map_launcher#128]
Hello 

Facing an issue on compile time

```
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.5.20 and higher.
The following dependencies do not satisfy the required version:
project ':map_launcher' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.50
```

Please provide solution, thanks.

**Comment by mattermoran** [comment]
I just bumped kotlin version 1.5.31 and released on pub as v2.4.0. Give it a try and let me know if it works :)

**Comment by mattermoran** [comment]
Closing as duplicate of https://github.com/mattermoran/map_launcher/issues/118
